### PR TITLE
Elastic indices 2

### PR DESCRIPTION
**Pull request by miiu96**
#### Description of the pull request (what is new / what has changed)

- Added documentation for Elasticsearch indices 

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/indices/accounts.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+id: es-index-accounts
+title: Elasticsearch index: accounts
+---
+
+The `_id` field for this index is composed of bech32 encoded address.
+
+| Field      | Description                                                                                                |
+|------------|------------------------------------------------------------------------------------------------------------|
+| address    | The address field holds the address bytes in a bech32 encoding.                                            |
+| balance    | The balance field holds the amount of EGLD the address has.                                                |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accounts.md:11]: The balance field holds the amount of EGLD the address possesses. It is a string that also includes the number of decimals. Example: `"1500000000000000000"` (equivalent to 1.5 EGLD).

File: docs/sdk-and-tools/indices/accounts.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| balanceNum | The balanceNum field holds the amount of EGLD the address has in a numeric format(denominated with 10^18). |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accounts.md:12]: The balance field holds the amount of EGLD the address possesses, in a numeric format. Example: `1.5`

File: docs/sdk-and-tools/indices/accounts.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+| nonce      | The nonce field represents the sequence number of the address.                                             |
+| shardID    | The shardId field represents in which shard the address belongs.                                           |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accounts.md:14]: The shardID field represents the shard the address belongs to, based on its bytes.

File: docs/sdk-and-tools/indices/accounts.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+| timestamp  | The timestamp field represents the timestamp when the address balance was changed.                         |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accounts.md:15]: The timestamp field represents the last moment when the address balance was changed.

File: docs/sdk-and-tools/indices/accounts.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            "balanceNum": {
+                "order": "desc"
+            }
+        }
+    ],
+    "size":10
+}'
+```
+
+### Fetch addresses sorted by balance from a shard
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accounts.md:37]: Fetch addresses in a shard, sorted by balance

File: docs/sdk-and-tools/indices/accountshistory.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+id: es-index-accountshistory
+title: Elasticsearch index: accountshistory
+---
+
+The `_id` field of this index is composed in this way: `{bech32address}_{timestamp}` (example: `erd.._1234`).
+
+| Field     | Description                                                                            |
+|-----------|----------------------------------------------------------------------------------------|
+| address   | The address field holds the address bytes in a bech32 encoding.                        |
+| balance   | The balance field holds the amount of EGLD the address has.                            |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accountshistory.md:11]: apply the same suggestions for the `address` and `balance` fields' description

File: docs/sdk-and-tools/indices/accountshistory.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+| isSender  | The isSender field is true if the address was the sender when the balance has changed. |
+| timestamp | The timestamp field represents the timestamp when the address balance was changed.     |
+
+## Query examples
+
+### Fetch the latest 10 recordings for an address sorted by timestamp
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accountshistory.md:17]: recordings -> entries

File: docs/sdk-and-tools/indices/blocks.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ id: es-index-blocks
 title: Elasticsearch index: blocks
 ---
 
-The `_id` field of this index is represented by the block hash, in a hexadecimal encoding.
+The `_id` field for this index is composed of hex encoded block hash.
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/blocks.md:6]: revert the change. the description was better before

File: docs/sdk-and-tools/indices/miniblocks.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+---
+id: es-index-miniblocks
+title: Elasticsearch index: miniblocks
+---
+
+The `_id` field for this index is composed of hex encoded miniblock hash.
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/miniblocks.md:6]: The `_id` field of this index is represented by the miniblock hash, in a hexadecimal encoding.

File: docs/sdk-and-tools/indices/miniblocks.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| senderShard       | The senderShard field represents the shard ID of the source block.                                                                                                     |
+| receiverShard     | The receiverShard field represents the shard ID of the destination block.                                                                                              |
+| senderBlockHash   | The senderBlockHash field represents the hash(hexadecimal encoded) of the source block in which the miniblock was included.                                            |
+| receiverBlockHash | The receiverBlockHash field represents the hash(hexadecimal encoded) of the destination block in which the miniblock was included.                                     |
+| type              | The type field represents the type of the miniblock. It can be `TxBlock`(if contains transactions) or `SmartContractResultBlock`(if contains smart contracts results). |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/miniblocks.md:15]: if contains -> if it contains

File: docs/sdk-and-tools/indices/miniblocks.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+| procTypeS         | The procTypeS field represents the processing type at the source shard. It can be `Normal` or `Scheduled`.                                                             |
+| procTypeD         | The procTypeD field represents the processing type at the destination shard. It can be `Normal` or `Scheduled`.                                                        |
+| timestamp         | The timestamp field represents the timestamp of the block in which the miniblock was executed.                                                                         |
+| reserved          | The reserved field ensures the possibility to extend the mini block.                                                                                                   |
+
+## Query examples
+
+### Fetch all the miniblocks from a block
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/miniblocks.md:23]: Fetch all the miniblocks of a block

File: docs/sdk-and-tools/indices/receipts.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+---
+id: es-index-receipts
+title: Elasticsearch index: receipts
+---
+
+The `_id` field for this index is composed of hex encoded receipt hash.
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/receipts.md:6]: for -> of

File: docs/sdk-and-tools/indices/rounds.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+| signersIndexes   | The signersIndexes is an array that contains the indices of the validators that should sign the block from this round. |
+| blockWasProposed | The blockWasProposed is true if a block was produced in this round.                                                    |
+| shardId          | The shardId field represents the shard the round belongs to.                                                           |
+| epoch            | The epoch field represents the epoch the round belongs to.                                                             |
+| timestamp        | The timestamp field represents the timestamp of the round.                                                             |
+
+
+## Query examples
+
+## Fetch the latest rounds for a shard when block was produced
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/rounds.md:21]: `##` ->  `###`

File: docs/sdk-and-tools/indices/scresults.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+| sender            | The sender field represents the address of the smart contract result sender.                                                                                   |
+| receiver          | The receiver field represents the destination address of the smart contract result.                                                                            |
+| senderShard       | The senderShard field represents the shard ID of the sender address.                                                                                           |
+| receiverShard     | The receiverShard field represents the shard ID of the receiver address.                                                                                       |
+| relayerAddr       | The relayerAddr field represents the address of the relayer.                                                                                                   |
+| relayedValue      | The relayedValue field represents the amount of EGLD to be sent from the relayed.                                                                              |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/scresults.md:22]: I think this is wrong. This field represents the amount of EGLD to be transferred via the inner transaction's sender, isn't it?

File: docs/sdk-and-tools/indices/scresults.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+| esdtValues        | The esdtValues field contains a list of ESDT values that are transferred based on the data field.                                                              |
+| receivers         | The receivers field contains a list of receiver addresses in case of ESDTNFTTransfer or MultiESDTTransfer.                                                     |
+| receiversShardIDs | The receiversShardIDs field contains a list of receiver addresses shard IDs.                                                                                   |
+| operation         | The operation field represents the operation of the smart contract result based on the data field.                                                             |
+| function          | The function field holds the name of the function that is called in case of a smart contract call.                                                             |
+| originalSender    | The originalSender field holds the address of the original sender.                                                                                             |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/scresults.md:38]: The originalSender field holds the sender's address of the original transaction.

File: docs/sdk-and-tools/indices/scresults.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+| timestamp         | The timestamp field represents the timestamp of the block in which the smart contract result was executed.                                                     |
+| status            | The status field holds the execution state of the smart contract result. The execution state can be `pending` or `success`.                                    |
+| tokens            | The tokens field contains a list of ESDT tokens that are transferred based on the data field.                                                                  |
+| esdtValues        | The esdtValues field contains a list of ESDT values that are transferred based on the data field.                                                              |
+| receivers         | The receivers field contains a list of receiver addresses in case of ESDTNFTTransfer or MultiESDTTransfer.                                                     |
+| receiversShardIDs | The receiversShardIDs field contains a list of receiver addresses shard IDs.                                                                                   |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/scresults.md:35]: addresses'

File: docs/sdk-and-tools/indices/scresults.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+| data              | The data field holds additional information for a smart contract result. It can contain a simple message, a function call, an ESDT transfer payload, or so on. |
+| prevTxHash        | The prevTxHash holds the hex encoded hash of the previous transaction.                                                                                         |
+| originalTxHash    | The originalTxHash holds the hex encoded hash of the transaction that generated the smart contract result.                                                     |
+| callType          | The callType field holds the type of smart contract call that is done through the smart contract result.                                                       |
+| codeMetaData      | The codeMetaData field holds the code metadata.                                                                                                                |
+| returnMessage     | The returnMessage field holds the message that is returned by a smart contract in case of error.                                                               |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/scresults.md:29]: an error

File: docs/sdk-and-tools/indices/accounts.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
+---
+id: es-index-accounts
+title: Elasticsearch index: accounts
+---
+
+The `_id` field for this index is composed of bech32 encoded address.
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:6]: done

File: docs/sdk-and-tools/indices/accounts.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+
+| Field      | Description                                                                                                |
+|------------|------------------------------------------------------------------------------------------------------------|
+| address    | The address field holds the address bytes in a bech32 encoding.                                            |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:10]: done

File: docs/sdk-and-tools/indices/accounts.md (hunk 1 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:11]: done

File: docs/sdk-and-tools/indices/accounts.md (hunk 2 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:12]: done

File: docs/sdk-and-tools/indices/accounts.md (hunk 3 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:14]: done

File: docs/sdk-and-tools/indices/accounts.md (hunk 4 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:15]: done

File: docs/sdk-and-tools/indices/accounts.md (hunk 5 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:37]: done

File: docs/sdk-and-tools/indices/accountshistory.md (hunk 6 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accountshistory.md:11]: done

File: docs/sdk-and-tools/indices/accountshistory.md (hunk 7 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accountshistory.md:17]: done

File: docs/sdk-and-tools/indices/miniblocks.md (hunk 9 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/miniblocks.md:6]: done

File: docs/sdk-and-tools/indices/miniblocks.md (hunk 10 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/miniblocks.md:15]: done

File: docs/sdk-and-tools/indices/miniblocks.md (hunk 11 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/miniblocks.md:23]: done

File: docs/sdk-and-tools/indices/rounds.md (hunk 13 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/rounds.md:21]: changed

File: docs/sdk-and-tools/indices/scresults.md (hunk 14 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/scresults.md:22]: yes, you are right. changed.

File: docs/sdk-and-tools/indices/scresults.md (hunk 15 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/scresults.md:38]: done

File: docs/sdk-and-tools/indices/scresults.md (hunk 16 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/scresults.md:35]: done.

File: docs/sdk-and-tools/indices/rounds.md [hunk 20]
```diff
@@ -0,0 +1,42 @@
...
+title: Elasticsearch index: rounds
+---
+
+The `_id` field of this index is composed in this way: `{shardID}_{round}` (example: `2_10905514`)
+
+
+| Field            | Description                                                                                                            |
+|------------------|------------------------------------------------------------------------------------------------------------------------|
+| round            | The round field represents the number of the round.                                                                    |
+| signersIndexes   | The signersIndexes is an array that contains the indices of the validators that should sign the block from this round. |
```
Comment (mariusmihaic) [inline docs/sdk-and-tools/indices/rounds.md:12]: should sign or did sign?

File: docs/sdk-and-tools/indices/rounds.md [hunk 21]
```diff
@@ -0,0 +1,42 @@
...
+| blockWasProposed | The blockWasProposed is true if a block was produced in this round.                                                    |
```
Comment (mariusmihaic) [inline docs/sdk-and-tools/indices/rounds.md:13]: [comment]

Flag name is `Proposed`, while comment says `produced`

File: docs/sdk-and-tools/indices/scresults.md [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+| code              | The code holds the code of the smart contract result.                                                                                                          |
+| data              | The data field holds additional information for a smart contract result. It can contain a simple message, a function call, an ESDT transfer payload, or so on. |
+| prevTxHash        | The prevTxHash holds the hex encoded hash of the previous transaction.                                                                                         |
+| originalTxHash    | The originalTxHash holds the hex encoded hash of the transaction that generated the smart contract result.                                                     |
+| callType          | The callType field holds the type of smart contract call that is done through the smart contract result.                                                       |
+| codeMetaData      | The codeMetaData field holds the code metadata.                                                                                                                |
```
Comment (mariusmihaic) [inline docs/sdk-and-tools/indices/scresults.md:28]: Perhaps add an additional explanation here?

File: docs/sdk-and-tools/indices/scresults.md [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+| returnMessage     | The returnMessage field holds the message that is returned by a smart contract in case of an error.                                                            |
+| timestamp         | The timestamp field represents the timestamp of the block in which the smart contract result was executed.                                                     |
+| status            | The status field holds the execution state of the smart contract result. The execution state can be `pending` or `success`.                                    |
+| tokens            | The tokens field contains a list of ESDT tokens that are transferred based on the data field.                                                                  |
+| esdtValues        | The esdtValues field contains a list of ESDT values that are transferred based on the data field.                                                              |
```
Comment (mariusmihaic) [inline docs/sdk-and-tools/indices/scresults.md:33]: Can you also add here that the indexes of `tokens` are linked to indexes in `esdtValues` list?

File: docs/sdk-and-tools/elastic-search.md [hunk 24]
```diff
@@ -87,26 +87,26 @@ Each entry in an Elasticsearch index will have a format similar to this:
...
-| [tags](/)                                                    | Contains the NFTs' tags.                                                    |
-| [logs](/)                                                    | Contains all the logs generated by transactions and smart contract results. |
-| [delegators](/)                                              | Contains details about all the delegators.                                  |
-| [operations](/)                                              | Contains all the transactions and smart contract results.                   |
-| [collections](/)                                             | Contains all the collections.                                               |
+| Name                                                               | Description                                                                 |
+|--------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| [transactions](/sdk-and-tools/indices/es-index-transactions)       | Contains all the transactions.                                              |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/elastic-search.md:92]: done.

File: docs/sdk-and-tools/elastic-search.md [hunk 25]
```diff
@@ -87,26 +87,26 @@ Each entry in an Elasticsearch index will have a format similar to this:
...
+| [blocks](/sdk-and-tools/indices/es-index-blocks)                   | Contains all the executed blocks.                                           |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/elastic-search.md:93]: done

File: docs/sdk-and-tools/elastic-search.md [hunk 26]
```diff
@@ -87,26 +87,26 @@ Each entry in an Elasticsearch index will have a format similar to this:
...
+| [validators](/sdk-and-tools/indices/es-index-validators)           | Contains the public keys of the validators grouped by epoch and shard.      |
+| [rating](/sdk-and-tools/indices/es-index-rating)                   | Contains the validators' rating for every epoch.                            |
+| [miniblocks](/sdk-and-tools/indices/es-index-miniblocks)           | Contains all the executed minblocks.                                        |
+| [rounds](/sdk-and-tools/indices/es-index-rounds)                   | Contains details of each round that has passed.                             |
+| [accounts](/sdk-and-tools/indices/es-index-accounts)               | Contains the addresses' balances and the timestamp when they were modified. | 
+| [accountshistory](/sdk-and-tools/indices/es-index-accountshistory) | Contains historical information about the address balances.                 |
+| [receipts](/sdk-and-tools/indices/es-index-receipts)               | Contains all the generated receipts.                                        |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/elastic-search.md:100]: done

File: docs/sdk-and-tools/elastic-search.md [hunk 27]
```diff
@@ -87,26 +87,26 @@ Each entry in an Elasticsearch index will have a format similar to this:
...
+| [scresults](/sdk-and-tools/indices/es-index-scresults)             | Contains all the generated smart contract results.                          |
+| [accountsesdt](/)                                                  | Contains the addresses' ESDT balances.                                      |
+| [accountsesdthistory](/)                                           | Contains historical information about the address ESDT balances.            |
+| [epochinfo](/)                                                     | Contains the accumulated fees and the developer fees grouped by epochs.     |
+| [scdeploys](/)                                                     | Contains details about all the deployed smart contracts.                    |
+| [tokens](/)                                                        | Contains all the created ESDT tokens.                                       |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/elastic-search.md:106]: done

File: docs/sdk-and-tools/elastic-search.md [hunk 28]
```diff
@@ -87,26 +87,26 @@ Each entry in an Elasticsearch index will have a format similar to this:
...
+| [tags](/)                                                          | Contains the NFTs' tags.                                                    |
+| [logs](/)                                                          | Contains all the logs generated by transactions and smart contract results. |
+| [delegators](/)                                                    | Contains details about all the delegators.                                  |
+| [operations](/)                                                    | Contains all the transactions and smart contract results.                   |
+| [collections](/)                                                   | Contains all the collections.                                               |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/elastic-search.md:111]: done

File: docs/sdk-and-tools/elastic-search.md [hunk 29]
```diff
@@ -87,26 +87,26 @@ Each entry in an Elasticsearch index will have a format similar to this:
...
+| [accountsesdthistory](/)                                           | Contains historical information about the address ESDT balances.            |
+| [epochinfo](/)                                                     | Contains the accumulated fees and the developer fees grouped by epochs.     |
+| [scdeploys](/)                                                     | Contains details about all the deployed smart contracts.                    |
+| [tokens](/)                                                        | Contains all the created ESDT tokens.                                       |
+| [tags](/)                                                          | Contains the NFTs' tags.                                                    |
+| [logs](/)                                                          | Contains all the logs generated by transactions and smart contract results. |
+| [delegators](/)                                                    | Contains details about all the delegators.                                  |
+| [operations](/)                                                    | Contains all the transactions and smart contract results.                   |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/elastic-search.md:110]: done

File: docs/sdk-and-tools/indices/accounts.md [hunk 30]
```diff
@@ -0,0 +1,58 @@
...
+|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| address    | The address field holds the address in a bech32 encoding. Should be equal to the _id field.                                                                                          |
+| balance    | The balance field holds the amount of EGLD the address possesses. It is a string that also includes the number of decimals. Example: "1500000000000000000" (equivalent to 1.5 EGLD). |
+| balanceNum | The balanceNum field holds the amount of EGLD the address possesses, in a numeric format. Example: 1.5.                                                                              |
+| nonce      | The nonce field represents the sequence number of the address.                                                                                                                       |
+| shardID    | The shardID field represents the shard the address belongs to, based on its bytes.                                                                                                   |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:14]: done

File: docs/sdk-and-tools/indices/rounds.md (hunk 20 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/rounds.md:12]: should

File: docs/sdk-and-tools/indices/rounds.md (hunk 21 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/rounds.md:13]: changed.

File: docs/sdk-and-tools/indices/scresults.md (hunk 23 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/scresults.md:33]: added

**Review by bogdan-rosianu — DISMISSED** [review]

File: docs/sdk-and-tools/indices/scresults.md [hunk 31]
```diff
@@ -7,35 +7,35 @@ The `_id` field for this index is composed of hex encoded smart contract result
...
+| codeMetaData      | The codeMetaData field holds the code metadata.                                                                                                                                       |
+| returnMessage     | The returnMessage field holds the message that is returned by a smart contract in case of an error.                                                                                   |
+| timestamp         | The timestamp field represents the timestamp of the block in which the smart contract result was executed.                                                                            |
+| status            | The status field holds the execution state of the smart contract result. The execution state can be `pending` or `success`.                                                           |
+| tokens            | The tokens field contains a list of ESDT tokens that are transferred based on the data field. The indices from the `tokens` list are linked with the indices from `esdtValues` list.  |
```
Comment (mariusmihaic) [inline docs/sdk-and-tools/indices/scresults.md:32]:
```suggestion
| tokens            | The tokens field contains a list of ESDT tokens that are transferred based on the data field. The indices from the `tokens` list are linked to the indices from `esdtValues` list.  |
```

**Review by bogdan-rosianu — APPROVED** [review]

**Review by mariusmihaic — APPROVED** [review]
